internal/monitor/short_integration_test.go

```diff
@@ -163,7 +145,7 @@ func TestUnityShortIntegration(t *testing.T) {
 
 	log.Printf("Starting integration test")
 	godogOptions := godog.Options{
-		Format:        outputFormat,
+        Format:        "pretty,cucumber:unity-integration-cucumber-report.json",
```

Comment:
Just double checking -- you want to have the same `xxx-integration-cucumber-report.json` target file regardless of whether the full integration or this short integration test is run?

Reply:
Assuming one is run at a time, I think it would be ok. On a single run of the short or long integration test each of the godog tests should produce a report file. Once the integration test finishes those godog report files are used for test reporting purposes and not needed anymore.

Do you foresee an issue with this?

Reply:
It may not be an issue. I was thinking that if the test result files needed to be named differently for reporting purposes-- to distinguish between the short-integration test runs vs. the full-integration test runs.

Reply:
I understand what you mean now. You are correct that better naming can be used here. Updated.